gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -37,24 +37,28 @@ type MsgCall struct {
 	FuncName string   // Function name
 	Args     []string // Function arguments
 	Send     string   // Send amount
+	Noop     bool
```

Comment:
You should not add new `Noop` fields to existing messages. Instead, add a new `NoopMsg` message.

Reply:
Because `Call(cfg BaseTxCfg, msgs ...MsgCall)`or `Send(cfg BaseTxCfg, msgs ...MsgSend)` use their respective specific messages, I added a field for classification. If we define MsgNoop, we will need to change the other parameters to an interface

Reply:
@moul please take a look at the latest commit after your suggestion. Btw, i also modify something to look clearly